containers/docker-compose-production.yml

```diff
@@ -21,3 +21,12 @@ services:
       - /etc/passwd:/etc/passwd:ro
       - /etc/group:/etc/group:ro
     restart: unless-stopped
+  mailman:
+    build: .
```

Comment:
maybe it has to be `..` here?

Reply:
and in https://github.com/publiclab/plots2/blob/master/containers/docker-compose-production.yml#L20 also?